src/SchemaView.tsx

```diff
@@ -1,48 +1,57 @@
-import { Dictionary, ISchema } from '@stoplight/types';
+import { Dictionary } from '@stoplight/types';
 import { Box, Button, IBox } from '@stoplight/ui-kit';
-import dropRight = require('lodash/dropRight');
+import { JSONSchema4 } from 'json-schema';
 import * as React from 'react';
 import { MutedText } from './common/MutedText';
-import { renderSchema } from './renderers/renderSchema';
+import { Property } from './components/Property';
+import { useProperties } from './hooks/useProperties';
 import { useTheme } from './theme';
-import { buildAllOfSchema } from './util/buildAllOfSchema';
+import { SchemaTreeNode } from './types';
+import { isExpanded } from './utils/isExpanded';
 
-export interface ISchemaViewProps {
+export interface ISchemaView extends IBox {
   name?: string;
   defaultExpandedDepth?: number;
-  schemas: object;
-  schema: ISchema;
+  dereferencedSchema?: JSONSchema4;
+  schema: JSONSchema4;
   limitPropertyCount?: number;
   hideRoot?: boolean;
   expanded?: boolean;
-  hideInheritedFrom?: boolean;
   emptyText: string;
 }
 
-export interface ISchemaView extends ISchemaViewProps, IBox {}
-
 export const SchemaView: React.FunctionComponent<ISchemaView> = props => {
   const {
     defaultExpandedDepth = 1,
     emptyText,
     expanded = false,
-    hideInheritedFrom = false,
     hideRoot,
-    limitPropertyCount = 0,
+    limitPropertyCount,
     schema,
-    schemas = {},
+    dereferencedSchema,
+    originalSchema,
```

Comment:
Where is this used? Don't see it in `ISchemaView`

Reply:
I believe it's a leftover. I need that for #12. Will remove.